core-feature-pack/src/main/resources/modules/system/layers/base/org/jboss/as/cli/main/module.xml

```diff
@@ -74,6 +74,7 @@
         <module name="org.jboss.threads"/>
         <module name="org.jboss.vfs"/>
         <module name="org.picketbox" optional="true"/>
+        <module name="org.wildfly.common"/>
```

Comment:
Can you add this in thecli/pom.xml as well please?  I don't see any previous compile-time uses of this in the cli module codebase itself.  The org.wildfly.common stuff ends up in jboss-cli-client.jar anyway, probably because of a transitive dependency via controller-client. But I think it's better if we're explicit in the poms and not rely on transitive deps.

Reply:
Done.